base/applications/cmdutils/where/where.c

```diff
@@ -0,0 +1,482 @@
+/*
+ * PROJECT:     ReactOS WHERE command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Search executable files
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+#include <windows.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strsafe.h>
+#include <conutils.h>
+#include "strlist.h" // strlist_...
+#include "resource.h"
```

Comment:
```suggestion
#include <stdlib.h>
#include <string.h>
#include <windef.h>
#include <winbase.h>
#include <strsafe.h>
#include <conutils.h>
#include "strlist.h" // strlist_...
#include "resource.h"
```

Reply:
What is "string.h" used for?

Reply:
`<string.h>` is deleted. `<winnls.h>` for `GetDateFormat/GetTimeFormat` is added.